mtest/mscore/albums/smallPianoAlbumTest.msca

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<museScore version="3.01" full-version="4.0.0-Development">
+  <Album>
+    <name>My Album</name>
+    <Score>
+      <alias>Piano1</alias>
+      <path>/home/boop/Documents/GSoC_2020/mscore/MuseScore/mtest/mscore/albums/Piano1.mscx</path>
```

Comment:
Having these absolute path makes it difficult to transfer albums.
Is it planned to have albums as compressed folders, containing all scores etc., just like the mscz file is a compressed folder containing the score plus some other files?

Reply:
I am planning to have both absolute and relative paths (if one does not work, the other might save the day).

Reply:
How about a compressed file containing all the score of the album, together with the new <code>msca</code> file? But that can be done after the albums itself are finalized.